app/controllers/bags_controller.rb

```diff
@@ -7,41 +7,34 @@ def index
 
   def new
     @bag = Bag.new
-    @user = current_user
   end
 
   def create
     @bag = Bag.new(bag_params)
+    @bag.user = current_user
     if @bag.save!
-      flash[:success] = "Bag successfully created"
       redirect_to bags_path
```

Comment:
do we want to redirect to the newly created bag show page?

Reply:
Make sense